usaspending_api/references/management/commands/load_offices.py

```diff
@@ -48,14 +48,24 @@ def process_data(self):
     def broker_fetch_sql(self):
         return f"""
             SELECT
-                office_code,
-                office_name,
-                sub_tier_code,
-                agency_code,
-                contract_awards_office,
-                contract_funding_office,
-                financial_assistance_awards_office,
-                financial_assistance_funding_office
-            FROM
-                office
+                o.office_code,
+                o.office_name,
+                o.sub_tier_code,
+                o.agency_code,
+                o.contract_awards_office,
+                o.contract_funding_office,
+                o.financial_assistance_awards_office,
+                o.financial_assistance_funding_office
+            FROM office o
+
+            /* Begin left anti joins to ensure we are not loading any offices
+            that are not linked to transactions */
+            LEFT JOIN source_assistance_transaction sat
+            ON sat.awarding_office_code = o.office_code
+
+            LEFT JOIN source_procurement_transaction spt
+            ON spt.awarding_office_code = o.office_code
+
+            WHERE sat.awarding_office_code IS NOT NULL
+                OR spt.awarding_office_code IS NOT NULL
```

Comment:
Just making sure, joining it with the transaction tables makes sense but could easily duplicate the results significantly. Is that a problem here and if so, would it make sense to dedupe the results with a distinct/group by? Same issue with the recipients too.

Reply:
We'll figure this out in QA, I'll also run a test to ensure that it doesn't. Our automated tests do not indicate that any records are duplicated.